jdaviz/cli.py

```diff
@@ -113,4 +117,4 @@ def _main():
 
     main(args.filename, layout=args.layout, browser=args.browser, theme=args.theme,
          verbosity=args.verbosity, history_verbosity=args.history_verbosity,
-         hotreload=args.hotreload)
+         hotreload=args.hotreload, instrument=args.instrument)
```

Comment:
I know they're kwargs so order doesn't matter, but it bothers me that these are in a different order than defined in the function (instrument between layout and browser).